src/shareddoc.js

```diff
@@ -244,6 +242,11 @@ export const persistence = {
       opts.headers = new Headers({
         Authorization: [...new Set(auth)].join(','),
         'X-DA-Initiator': 'collab',
+        'If-Match': '*',
+      });
+    } else {
+      opts.headers = new Headers({
+        'If-Match': '*',
```

Comment:
We should also add the `X-DA-Initiator` header here otherwise da-admin will invalidate the document back on collab. (PUT requests that don't have the `X-DA-Initiator: collab` header will invalidate the document in collab as it was updated from the outside).